cmd/stack_validate_build.go

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+// Simple test for appsody build command. A future enhancement would be to verify the image that gets built.
+func TestBuild(projectDir string) error {
+
+	// appsody build
+
+	imageName := "testbuildimage"
```

Comment:
I think this should be something like `dev.local/nodejs` where `nodejs` is the stack name (current directory for now, but eventually should be pulled from stack.yaml)